esc/import.go

```diff
@@ -0,0 +1,33 @@
+package esc
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// Parse an import id using {project_id}:{id} structure
+func parseImportID(importId string) ([]string, error) {
+	result := strings.Split(importId, ":")
+
+	if len(result) != 2 {
+		return nil, fmt.Errorf("failed to parse import id")
```

Comment:
The user might get the import ID wrong initially so it could be helpful to include it in the error message:
```suggestion
		return nil, fmt.Errorf("Failed to parse import id. Please use the format `{project_id}:{resource_id}`")
```

```bash
$ terraform import eventstorecloud_network.network c5e4fbcbcv45qdammis0
eventstorecloud_network.network: Importing from ID "c5e4fbcbcv45qdammis0"...

Error: Failed to parse import id. Please use the format `{project_id}:{resource_id}`
```